src/LinAlg/hiopMatrixRajaDense.cpp

```diff
@@ -1376,6 +1376,70 @@ double hiopMatrixRajaDense::max_abs_value()
   return maxv;
 }
 
+/**
+ * @brief Returns the value of the element with the maximum absolute value.
+ * 
+ * @todo Consider using BLAS call (<D>LANGE)
+ */
+void hiopMatrixRajaDense::row_max_abs_value(hiopVector &ret_vec)
+{  
+  assert(ret_vec.get_size() == m());
+  ret_vec.setToZero();
+
+  auto& vec = dynamic_cast<hiopVectorRajaPar&>(ret_vec);
+  vec.copyFromDev();
+  double* vd = vec.local_data_host();
```

Comment:
Why copy to the host mirror here?

Reply:
vd is used outside the RAJA section. 
If I don't make a host mirror, line 1411 will return me a segment fault.

Reply:
I think this will be taken care of if you parallelize the outer instead of the inner loop.